pkgs/tools/inputmethods/keyfuzz/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "keyfuzz-${version}";
+  version = "0.2";
+
+  meta = with stdenv.lib; {
+    description = "Manipulate the scancode/keycode translation tables of keyboard drivers.";
```

Comment:
@zimbatm https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes
Dot at the end.

Please be more careful..